packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import { node } from 'prop-types';
+import { C_LUNAR } from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+const StyledListItem = styled.li`
+  border-bottom: 0.0625rem solid ${C_LUNAR};
+  padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```

Comment:
Nice to have clear separation imo
```suggestion

  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```